test/ui/starter-select.test.ts

```diff
@@ -558,11 +558,11 @@ describe("UI - Starter select", () => {
     expect(starterSelectUiHandler?.cursorObj.y).toBe(31);
 
     game.onNextPrompt("SelectStarterPhase", UiMode.CONFIRM, () => {
-      const handler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+      const handler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
       handler.processInput(Button.ACTION);
     });
     game.onNextPrompt("SelectStarterPhase", UiMode.SAVE_SLOT, () => {
-      const saveSlotSelectUiHandler = game.scene.ui.getHandler() as SaveSlotSelectUiHandler;
+      const saveSlotSelectUiHandler = game.scene.ui.getCurrentHandler() as SaveSlotSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const saveSlotSelectUiHandler = game.scene.ui.getCurrentHandler<SaveSlotSelectUiHandler>();
```